packages/server/src/tests/epics/epicSelection.ts

```diff
@@ -189,6 +189,20 @@ export interface Result {
     debug?: Debug;
 }
 
+export const banditNullHypothesisFilter: Filter = {
+    id: 'matchesOneOfBanditNullHypothesisTests',
+    test: (test, targeting): boolean => {
+        const fiftyFiftyChance = getRandomNumber('NULL_HYPOTHESIS', targeting.mvtId) % 2;
+        if (test.name === 'AB_TEST_NULL_HYPOTHESIS') {
```

Comment:
```suggestion
        if (test.name.includes('BANDIT_NULL_HYPOTHESIS_TEST_AB')) {
```
alternatively check for a substring, so we don't need to decide on the date before the code is merged